### ISSUE TITLE
[Enhancement] Automatically activate new passive trees and gear sets

### ISSUE THREAD

**Issue by aggixx** [Openarl/PathOfBuilding#434]
Current behavior for passive trees is that when I click "New", type a name, and then hit enter the dialogue looks like this:

![image](https://user-images.githubusercontent.com/510805/27793070-1cef94ae-5fc2-11e7-9ca3-59335e05f933.png)

I feel that the newly created tree should become the currently active one. There would be little point to create a new, empty tree just to go back to modifying a different one immediately after. Manually activating the new tree is a simple process but seems like a step that is unnecessary since this should be the users intent in 95%+ of situations.

The same idea can be applied to new gear sets.

**Comment by Openarl** [comment]
Are you aware that you can activate a tree/item set in the management popups just by double-clicking it? The effort required to do that is beyond trivial, so I wouldn't see this as much of an improvement.

Anyway, one of the principles I follow when designing UIs in the program is not to assume the user always wants one action to occur as a result of another specific action, as un-doing an action that the program took on your behalf feels worse than having to perform that action manually, provided that doing it manually is trivial; and of course as you mentioned it's even worse if the user doesn't notice the program's action and gets confused. 

Of course you are correct that in this case the user will virtually always want to activate the new tree. But the same isn't true for the Item Set manager; a common use of Item Sets is to equip items onto Animate Weapon/Guardian, and since you can drag items onto the set selector in the sidebar to equip items onto the minion, you don't need to activate the minion's item set even once after creating it (and in fact it would be rather annoying if it happened automatically). So I wouldn't make this change there, and so for the sake of consistency I'd like to leave the tree manager alone as well.

**Comment by aggixx** [comment]
> Are you aware that you can activate a tree/item set in the management popups just by double-clicking it? The effort required to do that is beyond trivial, so I wouldn't see this as much of an improvement.

Yeah, I'm aware, although I didn't know that originally and the new tree not being activated left me a bit a confused the first couple times, especially in regards to copying (since I would name the trees appropriately and then mistakenly modify the wrong one).

> Of course you are correct that in this case the user will virtually always want to activate the new tree. But the same isn't true for the Item Set manager; a common use of Item Sets is to equip items onto Animate Weapon/Guardian, and since you can drag items onto the set selector in the sidebar to equip items onto the minion, you don't need to activate the minion's item set even once after creating it (and in fact it would be rather annoying if it happened automatically).

Ah neat, ok. Yeah I didn't know/think of that.

Thanks for explaining your thoughts.
